meta-freescale-3rdparty-mbl/conf/machine/imx6ul-des0258-mbl.conf

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+#@TYPE: Machine
+#@NAME: Technexion IMX6UL PICO-PI Development Board
+#@DESCRIPTION: Machine configuration for the PICO PI board
+
+MACHINEOVERRIDES =. "imx6ul-des0258:${MACHINE}:"
+
+include conf/machine/imx6ul-pico.conf
+require conf/include/mbl-default.conf
+
+# ------------------------------------------------------------------------------
+# non-MBL-specific (upstreamable) configuration
+# ------------------------------------------------------------------------------
+SERIAL_CONSOLES = "115200;ttymxc0"
+
+# ------------------------------------------------------------------------------
+# MBL-specific (non-upstreamable) configuration
+# ------------------------------------------------------------------------------
+
+
+##############################################################################
+# KERNEL symbol definitions for virtual/kernel build configuration
+#
+# KERNEL_CLASSES
+#   mbl-fitimage.bbclass is inherited into kernel.bbclass processing by
+#   defining this symbol to include mbl-fitimage.
+# KERNEL_IMAGETYPE
+#   The kernel is packages in a FIT image by specifying "fitImage"
+# KERNEL_DEVICETREE
+#   Its unnecessary to change this symbol here as the required
+#   "imx6l-pico.dtb" value is specified in imx6l-pico.conf (included above)
+##############################################################################
+#KERNEL_CLASSES  = "mbl-fitimage"
+KERNEL_DEVICETREE = "imx6ul-des0258.dtb"
+KERNEL_DEFCONFIG = "imx6ul_des0258_defconfig"
+
+INITRAMFS_IMAGE_BUNDLE = "1"
+IMAGE_BOOT_FILES += "zImage-initramfs-imx6ul-des0258-mbl.bin"
+
+# Bring in a boot script
+MACHINE_EXTRA_RRECOMMENDS_append = "mbl-boot-scr"
+IMAGE_BOOT_FILES += "boot.scr"
+
+##############################################################################
+# U-BOOT symbol definitions
+#
+# SPL_BINARY
+#   Specify the name of U-boot SPL binary. No SPL is needed if it is empty.
+##############################################################################
+#SPL_BINARY=""
+UBOOT_EXTLINUX = "0"
+#UBOOT_CONFIG = "pico-pi-imx6ul_defconfig"
+#UBOOT_MACHINE = "pico-pi-imx6ul_defconfig"
+#UBOOT_SUFFIX = "bin"
+#UBOOT_BINARY = "u-boot.${UBOOT_SUFFIX}"
+#UBOOT_ENTRYPOINT = "0x87800000"
+#UBOOT_DTB_LOADADDRESS = "0x83000000"
+#UBOOT_IMAGE = "mbl-u-boot.bin"
+#UBOOT_SIGN_ENABLE = "1"
```

Comment:
It looks like the code here is commented out, but there are some references to the PICO board here too as well. Should we remove them?